daily-challenges/2024-10/26/khanh5923_26.spec.ts

```
@@ -0,0 +1,47 @@
+import { test, expect, Page } from "@playwright/test";
+interface Product {
+  name: string | null;
+  qnt: string | null;
+}
+async function buyProduct(productsList: Product[], page: Page): Promise<void> {
+  let productSecondtList:Product[] = [];
+  for (const product of productsList) {
+    let visible = false;
+    try {
+      await expect(page.locator(`//div[contains(text(),'${product.name}')]`)).toHaveText(product.name!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)